Formula/h/hk.rb

```diff
@@ -32,14 +32,31 @@ def install
   end
 
   test do
+    assert_match version.to_s, shell_output("#{bin}/hk --version")
+
     (testpath/"hk.pkl").write <<~PKL
-      amends "https://hk.jdx.dev/v0/hk.pkl"
-      import "https://hk.jdx.dev/v0/builtins/cargo_clippy.pkl"
-      import "https://hk.jdx.dev/v0/builtins/cargo_fmt.pkl"
+      amends "package://github.com/jdx/hk/releases/download/v0.4.1/hk@0.4.1#/Config.pkl"
+      import "package://github.com/jdx/hk/releases/download/v0.4.1/hk@0.4.1#/builtins/cargo_clippy.pkl"
+      import "package://github.com/jdx/hk/releases/download/v0.4.1/hk@0.4.1#/builtins/cargo_fmt.pkl"
```

Comment:
Unless you want to use these below:
```suggestion
```

Reply:
oh I thought they are needed. I am okay to remove them